src/System.Collections/tests/Generic/HashSet/HashSet.Generic.Tests.cs

```diff
@@ -335,6 +335,35 @@ public void SetComparer_SequenceEqualTests()
 
         #endregion
 
+        #region TryGetValue
+
+        [Fact]
+        public void HashSet_Generic_TryGetValue_Contains()
+        {
+            T value = CreateT(1);
```

Comment:
Should probably test for at least one of reference types, primitive type and non-primitive value type. And for both default and custom comparers.